Getting company's role users and Email validation utility for UI

dadhich-ashish opened the pull request.

kmxo commented:
Thank you for your contribution. However, this submission has existing features from the server side API called Glide System (gs). 

For example: 

`gs.print('Department ID: ' + gs.getUser().getDepartmentID() + ' User Email: ' + gs.getUser().getEmail() + ' User is member of: ' + gs.getUser().getMyGroups());`

Closing this for now. Once you make additional changes, feel free to re-open this Pull Request or create a new one.

dadhich-ashish commented:
@kmxo  I've updated the existing file since it is relevant for the user. I did not add a new file because the previous code is also aligned with the userUtil.

Therefore, I enhanced the existing file and added new features to the script. Since this approach was accepted earlier, I added it there to avoid the risk of confusing developers with multiple files.

However, if you still have concerns, I can separate my code into a new file.
